### DISCUSSION TITLE
Change "hydro storage" to "pumped hydro"

### DISCUSSION THREAD

**Discussion by pbubik** [electricitymaps/electricitymaps-contrib#3311] [Ideas]
Could consider changing the term "hydro storage" to "pumped hydro"?

Hydro storage refers to the amount of energy stored behind a dam of a large hydro power project without having it pumped there. CA-BC for example has at any given time many TWh of storage on its massive reservoirs. It does not have a single pumped hydro energy storage project.

Pumped hydro is more accurate description of what we are tracking.

I appreciate an open discussion on this.

Peter

**Comment by perwijs** [comment]
Hello,
On the sources for production/consumption one sees the production of buffers like pumpstorage, but not when they are filled.
It would be interesting to see when the buffers are filled.  Are these data available ?
Kind regards,
Johan, Earth

**Reply by VIKTORVAV99** [reply]
This data is displayed when the source for the zone provides it. A good example is [El Hierro](https://app.electricitymaps.com/zone/ES-CN-HI).

**Comment by VIKTORVAV99** [comment]
I agree with you that it's probably a better description as normal hydro reservoirs are also technically hydro storage. Pumped storage is a more descriptive name as it indicates the plants can run in reverse as well (aka pumping up water into the reservoir).

@rasmusjeppsen what do you think?

**Comment by jarek** [comment]
See also https://github.com/electricitymaps/electricitymaps-contrib/issues/4552 where this suggestion was brought up again
